zuul.d/nodeset.yaml

```diff
@@ -309,7 +309,7 @@
       - name: compute-2
         label: cloud-centos-9-stream-tripleo
       - name: crc
-        label: coreos-crc-extracted-2-36-0-xxl
+        label: coreos-crc-extracted-2-36-0-3xl
```

Comment:
We try to maintain the name in sync with the labels used, in this case, the nodeset name says: `name: centos-9-medium-3x-centos-9-crc-extracted-2-36-0-xxl`. I am in favor of creating a new nodeset with a different name.

Reply:
done